terraform/modules/github-flux/data/flux.yaml

```diff
@@ -0,0 +1,186 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: ${namespace}
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels:
+    name: flux
+  name: flux
+  namespace: ${namespace}
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
```

Comment:
OK for now while we're testing. But as the flux helm operator only works in the context of a single namespace, we can narrow down the scope of these RBAC mappings (a role, rather than a clusterrole, may be more appropriate here).

Reply:
Had to move flux into it's own flux-system namespace as multiple flux helm operators did not work as expected.